src/components/Root/Root.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from 'styled-components';
+import theme from '../../theme';
+import '../../bootstrap'; // sets base CSS
+
+const Root = ({ children }) => (
```

Comment:
Choosing to use `Root` for the component name seems a little inconsistent with both `styled-components` and other component libraries like material-ui react. Since this is defaulting the theme provider I would suggest something like `BandwidthThemeProvider`. This would be similar to the root node found in the [react material ui component library](http://www.material-ui.com/#/get-started/usage) `MuiThemeProvider`